src/domains/idpay-app/api/idpay_qrcode_payment/io/openapi.qrcode_payment_io.yml.tpl

```diff
@@ -9,7 +9,7 @@ paths:
   /transaction/{transactionId}:
     get:
       tags:
-        - payment
+        - mergeIgnore
```

Comment:
is this API not needed by appio?

Reply:
no this API is not exposed to APPIO currently

Reply:
should it stay on a different swagger?

Reply:
i know this is correct, i don't know the reasons why it is in this swagger, if it is necessary we will delve into it separately